spec/json.coffee

```diff
@@ -0,0 +1,349 @@
+if typeof process isnt 'undefined' and process.execPath and process.execPath.indexOf('node') isnt -1
+  chai = require 'chai' unless chai
+  jsonParser = require '../lib/json'
+  fbpParser = require '../lib/fbp'
+  fbpParser.validateSchema = true # validate schema for every test on node.js. Don't have tv4 in the browser build
+else
+  fbpParser = require 'fbp'
+  jsonParser = require '../lib/json'
```

Comment:
Don't do a special require for the JSON parser. It should be exposed on the `fbp.serialize` (or similar).

Tests must test how external consumers are to use the functionality, and it is not right to expect them to dig inside node_modules/fbp/lib/json...